tensorflow/compiler/xla/service/gpu/stream_executor_util.cc

```diff
@@ -222,9 +222,13 @@ Status ExecuteKernelOnStream(const se::KernelBase& kernel,
 }
 
 se::GpuAsmOpts PtxOptsFromConfig(const HloModuleConfig& hlo_module_config) {
+  string extra_string = hlo_module_config.debug_options().xla_gpu_gpuasm_extra_flags();
+  std::vector<std::string> extra_flags;
+  extra_flags = absl::StrSplit(extra_string, " ", absl::SkipEmpty());
```

Comment:
Can you use comma delimiter? Using spaces makes it hard in mix with other options, e.g. XLA_FLAGS="--xla_gpu_gpuasm_extra_flags=a b c --xla_hlo_profile"